kafka-rest/src/main/java/io/confluent/kafkarest/ratelimit/FixedCostRateLimitFeature.java

```diff
@@ -77,4 +94,19 @@ private int getCost(ResourceInfo resourceInfo) {
     }
     return defaultCost;
   }
+
+  private static final class RequestRateLimiterCacheLoader
```

Comment:
We've got two of these now - the other one is in ProduceRateLimiters.

I wonder if we should pull it out to be in its own class.

No strong feelings though, you decide.

Reply:
@ehumber yeah, I was also thinking about that, my intention for putting this in private class because I thought the intention of the previous author is to hide this class `RequestRateLimiterCacheLoader`.

If you think it is necessary to do so to reduce code repetition, I'm happy to make it a standalone class.

Reply:
No strong feelings either way.  My only thought is, is this code likely to cause a problem/change/need updating, at which point there is a risk we forget to fix it in both places.  But I think that risk is low.

Reply:
I just saw, you did this already :)